src/web/src/components/Header.vue

```diff
@@ -14,12 +14,17 @@
       YACS
     </b-navbar-brand>
     <div>
-      <select v-model="selected" v-on:change="selectSemester(selected)">
-        <option value=""> Select Semester </option>
-        <option v-for="option in semesterOptions" v-bind:key="option.text">
+      <b-dropdown variant="outline-primary" 
+                        size = "sm" 
+                        text="Select Semester" 
```

Comment:
Instead of having it say `Select Semester` statically, we can change the text value dynamically to show what semester has been selected! I think this [link](https://stackoverflow.com/questions/56531990/vue-how-to-change-dropdown-text-properly/56533098) does the trick!

Reply:
Just updated it to dynamically change when you select a semester, but before setting the semester it says Select Semester (setting it to this.selectedSemester as the default value did not work).